taipy/gui/INSTALLATION.md

```diff
@@ -59,18 +62,18 @@ Open a new terminal and run the following commands:
 
 - Install the DOM dependencies
 ```bash
-cd gui
-cd dom
-npm i
+$ cd gui
```

Comment:
Can the user still copy/paste after the addition of the prompt?

Reply:
If we use the `$ `, we should use it everywhere. Otherwise, to let the user copy/paste the commands, we should remove all of these